devices.js

```diff
@@ -1790,6 +1790,24 @@ const devices = [
                 .withPreset(['schedule', 'manual', 'boost', 'complex', 'comfort', 'eco']),
         ],
     },
+    {
+        fingerprint: [{modelID: 'TY0202', manufacturerName: '_TZ1800_fcdjzz3s'}],
+        model: 'TY0202_occupancy',
+        vendor: 'TuYa',
+        supports: 'occupancy',
+        description: 'Motion sensor',
+        whiteLabel: [
+            {vendor: 'SilverCrest', model: 'SMSZ 1 A1'},
+        ],
+        fromZigbee: [fz.battery, fz.ias_occupancy_alarm_1],
+        toZigbee: [],
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
+            await bind(endpoint, coordinatorEndpoint, ['genPowerCfg']);
+        },
+        exposes: [e.battery(), e.occupancy()],
```

Comment:
`fz.ias_occupancy_alarm_1` potentially also supports, `e.low_battery()` and `e.tamper()`, can you check if this works?

Reply:
It works... I get an update 'false' for the low_battery and tamper is true, when I unclick the device. There's a small push-button on the back, which is pressed when the device is clicked in place. Then the tamper is false.

Reply:
I expect battery low to work, about the tamper usually it can be triggered by opening the device (e.g. to replace the battery). It should go to true after doing this.

Reply:
It certainly goes to false if tampered with. It's enough to unclick the device from its mount, then the button on the back is released, hence setting tamper to true and back to false when remounted.